CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 Bugfixes:
 
+* Faq: Fixed frontend JS which did not found if "yes" is checked.
```

Comment:
I'm sorry, I forgot a 4.0.2.
Ok, I'll remove it from changelog, don't you forget to add it when merging it.